docs/references/react/use-organization-list.mdx

```diff
@@ -1,3 +1,267 @@
 ---
-sanity_slug: /docs/reference/clerk-react/useorganizationlist
+title: useOrganizationList()
+description: Access methods for switching between active organizations and for listing available memberships, invitations and suggestions.
 ---
+
+import { Callout } from "nextra-theme-docs";
+import { Tables } from "@/components/Table";
+import { CodeBlockTabs } from "@/components/CodeBlockTabs";
+
+# `useOrganizationList()`
+
+Access methods for switching between active organizations and for listing available memberships, invitations and suggestions.
+
+`useOrganizationList` enables developers to fetch the memberships, invitations or suggestsions for an active user.
+
+`Note: these resources are paginated`
```

Comment:
I would use a Callout component here. But I think it would be better if it were placed under Returns, like this:

### Returns

The `userMemberships`, `userInvitations`, and `userSuggestions` are returned as paginated lists.

```suggestion
<Callout type="info">
  These resources are paginated.
</Callout>
```